app/views/workflows/_condition.html.erb

```diff
@@ -1,60 +1,47 @@
 <% case condition %>
 <% when Conditions::Conjunction %>
-  <table class="condition">
-    <tr>
-      <td rowspan="<%= condition.operations.size %>">AND</td>
-      <td><%= render_condition condition.operations[0] %></td>
-    </tr>
-    <% condition.operations[1..-1].each do |c| %>
-      <tr><td><%= render_condition c %></td></tr>
-    <% end %>
-  </table>
+  <span class="nested-condition">
+  <% condition.operations.each_with_index do |c, i| %>
+    <% if i > 0 %><span class="label label-success yields-text">and</span><% end %><%= render_condition c %><% end %>
+  </span>
 <% when Conditions::Disjunction %>
-  <table class="condition">
-    <tr>
-      <td rowspan="<%= condition.operations.size %>">OR</td>
-      <td><%= render_condition condition.operations[0] %></td>
-    </tr>
-    <% condition.operations[1..-1].each do |c| %>
-      <tr><td><%= render_condition c %></td></tr>
-    <% end %>
-  </table>
+  <span class="nested-condition">
+  <% condition.operations.each_with_index do |c, i| %>
+    <% if i > 0 %><span class="label label-success yields-text">or</span><% end %>
+    <%= render_condition c %>
+  <% end %>
+  </span>
 <% when Conditions::Negation %>
-  <table class="condition">
-    <tr>
-      <td>NOT</td>
-      <td><%= render_condition condition.operation %></td>
-    </tr>
-  </table>
+  <span class="nested-condition">
+    <span class="label label-danger yields-text">not</span>
+    <%= render_condition condition.operation %>
+  </span>
 <% when Conditions::Comparison %>
-  <table class="condition">
-    <tr>
-      <td rowspan="<%= condition.operations.size %>"><%= condition.type %></td>
-      <td><%= render_condition condition.operations[0] %></td>
-    </tr>
-    <% condition.operations[1..-1].each do |c| %>
-      <tr><td><%= render_condition c %></td></tr>
-    <% end %>
-  </table>
+  <span class="nested-condition">
+  <% condition.operations.each_with_index do |c, i| %>
+    <% if i > 0 %><span class="yields-text"><% case condition.type when
```

Comment:
this whole block needs some formatting for clarity, and ends should line up with if/do.